### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by Gbengacode**
### I made the following changes to this work 

- Added more schematic queries
- Added more tables
- Added more queries

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -40,4 +40,31 @@ ALTER TABLE ANIMALS ADD COLUMN OWNERS_ID int
...
+CREATE TABLE VETS (
+ID SERIAL PRIMARY KEY NOT NULL,
+NAME VARCHAR(100) NOT NULL,
+AGE INT NOT NULL,
+DATE_OF_GRADUATION DATE NOT NULL
+)
+
+CREATE TABLE specializations (
+  VID INT NOT NULL,
+  SID INT NOT NULL,
```
Comment (mmsesay) [inline schema.sql]: - [ ] Please use descriptive  names for your table columns. `VID` & `SID` aren't descriptive.  Why not use `species_id` and `vet_id` ?

File: data.sql [hunk 2]
```diff
@@ -228,3 +228,365 @@ SET OWNERS_ID =
...
+	 38, 
+	 '2008-06-08'
+ 
+ );
+
+-- Insert the following data for specialties
+
+ INSERT INTO SPECIALIZATIONS (
+ VID,
+ SID
```
Comment (mmsesay) [inline data.sql]: - [ ] Please use descriptive column names. `vet_id` and `species_id` are recommended.
Giving descriptive names will right away let the reviewer or other developers using your database know exactly what you are referring to.

Please update it in all the areas where you use `VID` and `SID`

File: data.sql [hunk 3]
```diff
@@ -228,3 +228,365 @@ SET OWNERS_ID =
...
+);
+
+
+
+
+
+-- Insert the following data for visits
+
+INSERT INTO VISITS (
+  AID,
```
Comment (mmsesay) [inline data.sql]: - [ ] Just like I mentioned above, please use descriptive column names.

File: data.sql [hunk 4]
```diff
@@ -228,3 +228,365 @@ SET OWNERS_ID =
...
+);
+
+INSERT INTO VISITS (
+  AID, 
+  VID,
+  VISITED_DATE
+)
+VALUES 
+(
+ (SELECT ID FROM ANIMALS WHERE NAME = 'Boarmon'),
```
Comment (mmsesay) [inline data.sql]: - [ ] The animal name for `Vet Stephanie Mendez` should be `Blossom` not `Boarmon`

File: data.sql [hunk 5]
```diff
@@ -228,3 +228,365 @@ SET OWNERS_ID =
...
```
Comment (mmsesay) [inline data.sql]: - [ ] The animal name for `Vet William Tatcher` should be `Blossom` not `Boarmon`

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Gbengacode, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @Gbengacode,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**